app/globals.css

```
@@ -1,23 +1,33 @@
-@import 'tailwindcss';
+@import 'tailwindcss' source(none);
 @import 'tw-animate-css';
 
-@custom-variant dark (&:is(.dark *));
-@custom-variant motion-reduce (@media (prefers-reduced-motion: reduce));
+/* Only scan shipped app routes and shared source code.
+   Legacy routes like /burn, /currency, /reserves, and /test-locale stay out
+   of the production CSS build unless they are explicitly reintroduced here. */
+@source "./page.tsx";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)